src/mqt/predictor/ml/notebooks/mqt_predictor_ML.ipynb

```diff
@@ -0,0 +1,662 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "5189fbc1",
+   "metadata": {},
+   "source": [
+    "# Data Preparation"
+   ]
+  },
+  {
```

Comment:
I'll just annotate this here, as I found no better place. The whole `notebooks` folder should _not_ be part of the distributed Python package.

I would have imagined the overall structure to be something like
```console
/src
  ... all the source code ...
/notebooks
  /ml (general directory for `ml` stuff)
    /results (directory for `ml` results)
  ..
```

Reply:
Done.